src/displayapp/screens/WeatherSymbols.h

```diff
@@ -6,7 +6,7 @@ namespace Pinetime {
   namespace Applications {
     namespace Screens {
       namespace Symbols {
-        const char* GetSymbol(const Pinetime::Controllers::SimpleWeatherService::Icons icon) {
+        static const char* GetSymbol(const Pinetime::Controllers::SimpleWeatherService::Icons icon) {
```

Comment:
Curious: what's the reasoning behind using a static variable in a namespace? Wouldn't this just mean that it is instanced into every compilation unit?

Reply:
Right, without it you end up re-defining it and the linker throws a `multiple definition` error.

Reply:
Yeah, this should be marked as static. It means that the function can be called without being tied to an object. This is different from declaring static linkage, which this function won't have.

Reply:
The best way to solve the issue of multiple definitions would probably be to declare the function in the .h file and create a .cpp file that implements the function.

Reply:
This is solved by #1969.